### PR TITLE
Update Bootcamp EDA: Instructions for reviewer & how to deal with missing values

### PR DESCRIPTION
**Pull request by rethabilethulo**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Andy-Nkumane — APPROVED** [review]

File: content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md [hunk 1]
```diff
@@ -50,4 +50,12 @@ The [data set](data.csv) consists of health and demographic data for the period
...
 
-10. Create appropriate graphs to visually represent the relationship between literacy and life expectancy by region, and then for African countries. What can be concluded from the graphs? How confident can we be in the relationships represented here?
\ No newline at end of file
+10. Create appropriate graphs to visually represent the relationship between literacy and life expectancy by region, and then for African countries. What can be concluded from the graphs? How confident can we be in the relationships represented here?
+
+
+### Instructions for reviewers
+
+- Ensure that correct methods were used to check for any missing values in the data, also check how they dealt with missing values and if they gave a valid reason(s) for choosing that method.
+- Ensure that they checked if the data has any other problems and fixed the problems if they detected any problems.
```
Comment (Samantha-Hampton) [inline content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md]: "Ensure that they checked if the data has any other problems and resolved them if so."

File: content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md [hunk 2]
```diff
@@ -50,4 +50,12 @@ The [data set](data.csv) consists of health and demographic data for the period
...
+- Ensure that the notebook has an introduction at the beginning of the notebook, insights for every answered question, and a conclusion at the end of the notebook.
```
Comment (Samantha-Hampton) [inline content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md]: "Ensure that the notebook has an introduction at the beginning, insights for every answered question, and a conclusion at the end."

File: content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md [hunk 3]
```diff
@@ -50,4 +50,12 @@ The [data set](data.csv) consists of health and demographic data for the period
...
+- Ensure that all plots are labeled and the title and axis are bold.
```
Comment (Samantha-Hampton) [inline content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md]: "Ensure that all plots have their titles and axes labelled in bold."

**Review by Samantha-Hampton — CHANGES_REQUESTED** [review]
Just some minor grammatical edits

**Review by Samantha-Hampton — APPROVED** [review]

**Review by ry-oc — APPROVED** [review]
looks good

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**